### ISSUE TITLE
can't ssh into device via usb

### ISSUE THREAD

**Issue by popanz** [sailfish-on-fxtecpro1/droid-config-t5#20]

**Comment by Kabouik** [comment]
Confirmed, even though USB developer mode is active (confirmed from Settings GUI) and the IP is not in the `~/.ssh/known_hosts` on the host. No problem over WIFI.

**Comment by Kabouik** [comment]
Not tested yet, but it is likely fixed in [4.2.1.20](https://together.jolla.com/question/217843/release-notes-321-nuuksio-released/):

> SSH over USB fixed

**Comment by Kabouik** [comment]
Confirmed, fixed on 4.2.1.20:

```
nemo@192.168.2.15's password: 
Last login: Thu Dec 26 02:36:42 2019
,---
| Sailfish OS 3.2.1.20 (Nuuksio)
'---
[nemo@Sailfish ~]$ 
```

**Comment by popanz** [comment]
Unfortunately I can't confirm. Connecting via USB sill don't work for me.

**Comment by Kabouik** [comment]
I can confirm I'm having the same issue @popanz reported again. Yet, it is surprising that it did work perfectly on Dec. 26 just after I upgraded to 3.2.1.20 (as shown in my post). Is it possible that somehow our SFOS port restored a bug that was just fixed by Jolla upstream? Or could it be related to the boot image?

MTP seems to fail too. The phone is detected in Nautilus, but I cannot access it:

![Screenshot from 2020-01-28 17:16:16](https://user-images.githubusercontent.com/7107523/73283438-64124a00-41f3-11ea-851e-111d843557cf.png)

**Comment by piggz** [comment]
I have a fix for this in #34

**Comment by 0312birdzhang** [comment]
> I have a fix for this in #34

It's working on Redmi K20 too, thank you.
BTW, i need wait about 2 minutes to make it works, and 2323 port hangs too in this 2 minutes, ssh over WiFi is okay.

**Comment by piggz** [comment]
This is now resolved
